src/families/filecoin/specs.ts

```diff
@@ -9,6 +9,7 @@ import type { AppSpec } from "../../bot/types";
 
 const MIN_SAFE = new BigNumber(100000);
 const filecoinSpecs: AppSpec<Transaction> = {
+  disabled: true, // Bot spec is currently not correctly working
```

Comment:
I don't think we have logs or warnings about disabled specs, right?
Could we have some during bot run?